app/views/articles/show.html.erb

```diff
@@ -1,10 +1,56 @@
+<h1>Article # <%= @article.id %></h1>
 <p>
-	<strong>Tittel:</strong>
+	<h3>Title:</h3>
 	<%= @article.title %>
 </p>
 <p>
-	<strong>Tekst:</strong>
+	<h3>Text:</h3>
 	<%= @article.text %>
 </p>
-<%= link_to 'Endre innlegget', edit_article_path %><br>
-<%= link_to 'Til oversikten', articles_path %>
\ No newline at end of file
+
+<!-- <% i = - @article.comments.count + 1 %> -->
+<% if @article.comments.count > 0 %>
+	<h2> - Comments:</h2>
+	<table>
+			<%= render @article.comments, :i => i %>
```

Comment:
It looks like maybe you were planning on using it to number the comments - I think actually I would use CSS for that. If you really wanted to do it in Ruby though, you can be more explicit and render each comment like this:

```erb
<% @article.comments.each do |comment| %>
  <%= render comment %>
<% end %>
```

and then you can replace `each` with `each_with_index` to also get the index in the loop:

```erb
<% @article.comments.each do |comment, index| %>
  <%= render comment, index: index %>
<% end %>
```

Again though, I think I would use CSS for this.